### PR TITLE
S49 main misdemeanor branch

### PR DESCRIPTION
**Pull request by LaraDei**

### CODE REVIEW

File: src/data/calculatorPages.js [hunk 1]
```diff
@@ -44,21 +44,33 @@ const BODY_TYPES = {
...
+    ],
+    buttons: [
+      {
+        text: "Start",
+        href: CALC_PREFIX + "/landing-1",
+      },
+    ],
+    tooltip: "My conviction is not a misdemeanor or I’m not sure.",
+  },
+  "landing-ineligible-0": {
```
Comment (dylankershaw) [inline src/data/calculatorPages.js:63]:
```suggestion
```

Looks like duplicate code was added by accident.

File: src/data/calculatorPages.js [hunk 2]
```diff
@@ -44,21 +44,33 @@ const BODY_TYPES = {
...
+    body: [
+      {
+        type: BODY_TYPES.PARAGRAPH,
+        text: "It looks like your conviction may not be a misdemeanor. Our calculator is unable to assist you with your conviction at this point. We would like to hear your input and learn about your experience as we continue building our calculator.",
+      },
+    ],
+    footerLink: {
+      text: "Take Survey",
+      href: "https://docs.google.com/forms/d/e/1FAIpQLSeuT2oIBnxFAMv6p8tsujNlSugWxFS5la4DUC4rTFekInrcqQ/viewform?usp=sf_link",
     },
```
Comment (dylankershaw) [inline src/data/calculatorPages.js:74]:
```suggestion
```

This is duplicate code. Might have been added accidentally during the merge conflict resolution.

**Review by dylankershaw — APPROVED** [review]

**Review by LaraDei — COMMENTED** [review]
Removed duplicate

**Comment by dylankershaw** [comment]
Looks like the build failed due to a syntax error from this merge.

**Pull request merged**